src/nccl_ofi_rdma.c

```diff
@@ -1509,9 +1509,10 @@ static inline int process_err_completion(nccl_net_ofi_rdma_ep_t *ep,
 		req = err_entry.op_context;
 	}
 
-	NCCL_OFI_WARN("Request %p completed with error. RC: %d. Error: %s. Completed length: %ld, Request: %s",
+	NCCL_OFI_WARN("Request %p completed with error. RC: %d. Error: %s (%d). Completed length: %ld, Request: %s",
```

Comment:
Let's format this as `RC: %d.  Error: %d (%s)`.  Much easier for customers to write log parsers for (and many write log parsers).